apps/rotm/acceptance/features/back-link-check-your-report.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
+
+const defaults = require('../defaults');
+
+Feature('Given I am on /image');
+
+// bugfix
```

Comment:
erm, what's this comment for? Remove it or elaborate substantially

Reply:
It was an approach that I picked up from Lenny in r2r.  See here, https://github.com/UKHomeOffice/right-to-rent-check/search?utf8=%E2%9C%93&q=bugfix&type=

If a new dev was to review this codebase for the first time, they may question why we have such a basic test.  Adding the comment would help to explain there was a bug previously